projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.ts

```diff
@@ -427,15 +419,19 @@ export class IgxDateRangePickerComponent extends DisplayDensityBase
     }
 
     private _value: DateRange;
-    private _collapsed = true;
+    private overlayId: string;
```

Comment:
you have now 6 private members like this`_withUnderScore` and four like this `noUnderScorre`
Please let make this consistent :)